packages/sourcecred/src/ui/components/LedgerViewer/LedgerEventRow.js

```diff
@@ -122,9 +145,11 @@ const LedgerEventRow = (props: LedgerEventRowProps): ReactNode => {
   return (
     <TableRow>
       <TableCell component="th" scope="row">
-        <Typography variant="button">
-          {props.event.action.type.replace("_", " ")}
-        </Typography>
+        <ThemeProvider theme={actionTheme}>
+          <Typography color="primary">
+            {_friendlyFormatActionType(props.event.action.type)}
```

Comment:
I would put this value within a `useMemo`

Reply:
Is the result of this purely a difference in performance and an observation of performance best practices or are there other differences in the results?

Reply:
Just a way to avoid memory clutter and unnecessary computations between re-renders not to recreate the function or recalculate the values especially if this is a list item where this would happen an n amount of times

Reply:
Updated.. I'm not sure what the right way to apply useMemo would have been within the content of Typography, but I did implement it in the function. It was easier for me to reason about what was being memoized and why.